pkg/ipam/subnet.go

```diff
@@ -321,6 +321,7 @@ func (subnet *Subnet) GetStaticAddress(podName, nicName string, ip IP, mac *stri
 					subnet.V4IPToPod[ip.String()] = fmt.Sprintf("%s,%s", subnet.V4IPToPod[ip.String()], podName)
 					return ip, macStr, nil
 				}
+				klog.Errorf("ip %s has been allocated to %s", ip.String(), pods)
```

Comment:
the type of pods is slice, which maybe can not be printed with %s

Reply:
ok %v is better